### PR TITLE
<SectionHelper/> - unify paddings with no close button

### PR DESCRIPTION
**Pull request by Zivilekin**
### 🔦 Summary

Curently Section Helper content has left padding set to 24px, and right padding to 42px at all times. Let's change it depending on a presence of a close button: 

    -keep padding as it is (left 24px, right 42px) when section helper has a close button
    -if close button is removed unify both sides padding to be 24px

### CODE REVIEW

**Comment by EgoziE** [comment]
Thanks for submitting this PR.
Please take the time to read how to properly create a PR for wix-style-react with signed commits.

https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6097

File: src/SectionHelper/test/SectionHelper.visual.js [hunk 1]
```diff
@@ -4,20 +4,47 @@ import { Layout, Cell } from '../../Layout';
...
+  ))
+  .add('Padding: with/without CloseButton', () => (
+    <div style={{ width: '250px' }}>
+      <Layout>
+        <Cell>
+          <SectionHelper
+            actionText="I understand the consequences"
+            title="Look at this really long and important message that could in some cases contain many lengthy words like psychophysicotherapeutics!"
+            children="This is the content of very important message which actually has a lot of detailed explanation about various things. It may even have multiple sentences but they do not need to be those boring Lorem Ipsum"
+            onAction={() => undefined}
```
Comment (argshook) [inline src/SectionHelper/test/SectionHelper.visual.js:33]:
```suggestion
            onAction={() => undefined}
```
probably not needed

File: src/SectionHelper/test/SectionHelper.visual.js [hunk 2]
```diff
@@ -4,20 +4,47 @@ import { Layout, Cell } from '../../Layout';
...
+            showCloseButton
+            onClose={() => undefined}
```
Comment (argshook) [inline src/SectionHelper/test/SectionHelper.visual.js:35]:
```suggestion
            onClose={() => undefined}
```
probably not needed

File: src/SectionHelper/test/SectionHelper.visual.js [hunk 3]
```diff
@@ -4,20 +4,47 @@ import { Layout, Cell } from '../../Layout';
...
+          />
+        </Cell>
+        <Cell>
+          <SectionHelper
+            actionText="I understand the consequences"
+            title="Look at this really long and important message that could in some cases contain many lengthy words like psychophysicotherapeutics!"
+            children="This is the content of very important message which actually has a lot of detailed explanation about various things. It may even have multiple sentences but they do not need to be those boring Lorem Ipsum"
+            onAction={() => undefined}
```
Comment (argshook) [inline src/SectionHelper/test/SectionHelper.visual.js:43]:
```suggestion
            onAction={() => undefined}
```
probably not needed

File: src/SectionHelper/test/SectionHelper.visual.js [hunk 4]
```diff
@@ -4,20 +4,47 @@ import { Layout, Cell } from '../../Layout';
...
+            showCloseButton={false}
+            onClose={() => undefined}
```
Comment (argshook) [inline src/SectionHelper/test/SectionHelper.visual.js:45]:
```suggestion
            onClose={() => undefined}
```
probably not needed

File: src/SectionHelper/test/SectionHelper.visual.js [hunk 5]
```diff
@@ -4,20 +4,47 @@ import { Layout, Cell } from '../../Layout';
...
+      ))}
+    </Layout>
+  ))
+  .add('Padding: with/without CloseButton', () => (
+    <div style={{ width: '250px' }}>
+      <Layout>
+        <Cell>
+          <SectionHelper
+            actionText="I understand the consequences"
+            title="Look at this really long and important message that could in some cases contain many lengthy words like psychophysicotherapeutics!"
```
Comment (argshook) [inline src/SectionHelper/test/SectionHelper.visual.js:31]: `actionText` and `title` are the same in both example, would be nice to have this content only once in a variable

**Review by argshook — CHANGES_REQUESTED** [review]

File: src/SectionHelper/test/SectionHelper.visual.js (hunk 2 above)
Comment (daugerdas) [reply src/SectionHelper/test/SectionHelper.visual.js]: seems that CloseButton is not displayed then

File: src/SectionHelper/test/SectionHelper.visual.js (hunk 4 above)
Comment (daugerdas) [reply src/SectionHelper/test/SectionHelper.visual.js]: seems that CloseButton is not displayed then

**Review by argshook — APPROVED** [review]
great work! :+1:

**Pull request merged**